test/activation_based/test_precision_fp8_hardware.py

```
@@ -0,0 +1,560 @@
+                atol=0.25,
+                rtol=0.35,
+            )
+            _assert_gradient_quality(
+                candidate_gradient,
+                reference_gradient,
+                min_cosine=gradient_min_cosine,
+                max_relative_l2=gradient_max_relative_l2,
+            )
+    assert any(
+        not torch.equal(previous, parameter.detach())
+        for previous, parameter in zip(before, model.parameters())
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)